vignettes/EIRprevmatch.Rmd

```diff
@@ -89,85 +173,209 @@ prev <- lapply(
 
-points(x=EIR_prev$init_EIR, 
-       y=EIR_prev$prev,
+# Overlay the initial EIR and corresponding PfPR2-10 points from malariasimulation
+points(x = malSim_P2E$init_EIR, 
+       y = malSim_P2E$prev,
        pch = 19,
        col = 'black')
 
-lines(x=fit$init_EIR, 
-      y=fit$fit, 
-      col = "red", 
+# Overlay the malariasimulation line of best fit:
+lines(x = malSim_fit$init_EIR, 
+      y = malSim_fit$malSim_fit, 
+      col = plot_cols[1],
+      lwd = 2,
       type = "l", 
       lty = 1)
 
-lines(x=prevmatch$eir, 
-      y=prevmatch$prev, 
-      col = "blue", 
-      type = "l", 
+# Overlay the malariaEquilibrium Initial EIR to PfPR2-10 line
+lines(x = malEq_P2E$EIR, 
+      y = malEq_P2E$prev, 
+      col = plot_cols[2], 
+      type = "l",
+      lwd = 2,
       lty = 1)
 
-legend("bottomright", legend=c("malariasimulation", "malariaEquilibrium"), col=c("red", "blue"), lty = c(1,1), cex=0.8)
+# Add a legend
+legend("topright",
+       legend = c("malariasimulation", "malariaEquilibrium"),
+       col = c(plot_cols[1:2]), lty = c(1, 1), cex=0.8, box.col = "white")
 
 ```
 
-Now extract values from the fitted line to generate EIR estimates at your desired PfPR2-10 values.
+Looking at the plot, we can see that, to achieve a given *Pf*PR~2-10~, the `malariaEquilibrium` and `malariasimulation` approaches recommend a quite different EIR value, with the `malariaEquilibrium` method typically requiring a much lower EIR value to achieve a given *Pf*PR~2-10~ (given the `malariasimulation` parameter set used).
+
+## Matching EIR to *Pf*PR~2-10~ values
+
+Using the fitted relationship between initial EIR and *Pf*PR~2-10~, we can create a function that returns the EIR value(s) estimated to yield a target *Pf*PR~2-10~ given the model parameters. The function below works by finding the closest *Pf*PR~2-10~ value from the values generated when we fit the model to the target value input by the user, and then using that index to return the corresponding initial EIR value.
 
 ```{r table, warning=F}
-# Pre-intervention baseline PfPR2-10 starting at values 10, 25, 35, 55 
-PfPR <- c(.10, .25, .35, .45)
 
-# match via stat_smooth predictions
-match <- function(x){
+# Store some pre-intervention baseline PfPR2-10 values
+PfPRs_to_match <- c(.10, .25, .35, .45)
+
+# Create a function to these baseline PfPR2-10 values to EIR values using
+# our model fit:
+match_EIR_to_PfPR <- function(x){
   
-  m <- which.min(abs(fit$fit-x)) # index of closest prev match
-  fit[m,2]                       # print match
+  m <- which.min(abs(malSim_fit$malSim_fit-x))
+  malSim_fit[m,2]
   
 }
 
-eir <- unlist(lapply(PfPR, match))
+# Use the function to extract the EIR values for the specified 
+# PfPR2-10 values:
+matched_EIRs <- unlist(lapply(PfPRs_to_match, match_EIR_to_PfPR))
+
+# Create a dataframe of matched PfPR2-10 and EIR values:
+cbind.data.frame(PfPR = PfPRs_to_match, Matched_EIR = matched_EIRs)
+
+```
+
+## Calibrating *Pf*PR~2,10~ using the cali package
```

Comment:
Let's think about leaving `cali` out as a dependency and just putting in the final graph here as an image (similar to what Richard did in the model vignette). You could leave in your explanations and even your code, just don't have R run that block when building the .Rmd.

Reply:
Done! Also added a short paragraph to the intro letting the readers/users know we've done this and that they can experiment on their by uncommenting these lines.